test/base.test.js

```diff
@@ -60,8 +60,7 @@ test('should support multiple secrets', async (t) => {
   const plugin = fastifyPlugin(async (fastify, opts) => {
     fastify.addHook('onRequest', (request, reply, done) => {
       request.sessionStore.set(sessionId, {
-        sessionId,
-        encryptedSessionId: sessionIdSignedWithOldSecret
+        test: {}
```

Comment:
For accuracy -- `sessionId` / `encryptedSessionId` should never be in the session store anymore